modules/contacts/controller.js

```diff
@@ -2,17 +2,18 @@ const { isValidObjectId } = require("mongoose");
 const contactsService = require("./service");
 
 const listContacts = async (req, res, next) => {
-  let contacts = await contactsService.getAll();
+  const contacts = await contactsService.getAll();
   const { page, limit, favorite } = req.query;
+  let filteredContacts = contacts;
```

Comment:
OH, GOD, no!! 🙈 To tworzy referencje (!!!). ANTYWZOR :) Bad. bad. Zrob `[]` zamiast `contacts` w tej linijce.